### DISCUSSION TITLE
Active Links

### DISCUSSION THREAD

**Discussion by Turquioii** [ScratchAddons/ScratchAddons#3664] [Ideas] [answered]
A lot of links left in projects, aren't from scratch, and they are quite the pain to open manually. So the links should be highlighted, like the default Scratch ones. And when clicked do the following:
Click ->
If not from scratch: Prompt that you are leaving website.
Redirect

**Comment by cobaltt7** [comment, marked as answer]
“More links” addon. Implemented
